node/resolvers/search/index.ts

```diff
@@ -401,13 +403,15 @@ export const queries = {
 
     const [regionId, selectedFacets] = getRegionIdFromSelectedFacets(args.selectedFacets)
 
-    const sellers = await getSellers(vbase, checkout, segment?.channel, regionId || segment?.regionId)
+    const tradePolicy = getTradePolicyFromSelectedFacets(selectedFacets) || segment && segment.channel
```

Comment:
```suggestion
    const tradePolicy = getTradePolicyFromSelectedFacets(selectedFacets) || segment?.channel
```
just a suggestion to improve readability :)